In mobile view, main site top right menu button opens main nav bar instead of docs

martinxu9 opened the pull request:
In mobile view, main site top right menu button opens slightly modified main nav bar instead of docs.
<img width="649" alt="image" src="https://github.com/reflex-dev/reflex-web/assets/15661672/b4fdc9f0-fb92-4cf7-a695-61203973e04e">

picklelo reviewed (commented):
The text is a bit dark on the sidebar, can we make it brighter? Everything else looks good to me

martinxu9 commented:
> The text is a bit dark on the sidebar, can we make it brighter? Everything else looks good to me

changed to pure white, updated the screenshot in PR summary
